src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExportSamples.tsx

```diff
@@ -0,0 +1,303 @@
+          name={["samples", sample.name, "libraryStrategy"]}
+          label={i18n("NcbiBioSample.libraryStrategy")}
+        >
+          <Select style={{ display: "block" }} onChange={onChange}>
+            {strategies?.map((option: string) => (
+              <Select.Option key={option}>{option}</Select.Option>
+            ))}
+          </Select>
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item
+          rules={rules}
+          name={["samples", sample.name, "librarySource"]}
+          label={i18n("NcbiBioSample.librarySource")}
+        >
+          <Select style={{ display: "block" }} onChange={onChange}>
+            {sources.map((option: string) => (
+              <Select.Option key={option}>{option}</Select.Option>
+            ))}
+          </Select>
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item
+          rules={rules}
+          name={["samples", sample.name, "libraryConstructionProtocol"]}
+          label={i18n("NcbiBioSample.libraryConstructionProtocol")}
+        >
+          <Input type="text" onChange={onChange} />
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item
+          rules={rules}
+          name={["samples", sample.name, "instrumentModel"]}
+          label={i18n("NcbiBioSample.instrumentModel")}
+        >
+          <Cascader
+            options={platforms}
+            style={{ display: "block" }}
+            onChange={onChange}
+          />
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item
+          rules={rules}
+          name={["samples", sample.name, "librarySelection"]}
+          label={i18n("NcbiBioSample.librarySelection")}
+        >
+          <Select style={{ display: "block" }} onChange={onChange}>
+            {selections.map((option) => (
+              <Select.Option key={option}>{option}</Select.Option>
+            ))}
+          </Select>
+        </Form.Item>
+      </Col>
+      {sample.files.singles.length > 0 && (
+        <Col span={24}>
+          <Form.Item
+            name={["samples", sample.name, "files", "singles"]}
+            label={i18n("CreateNcbiExport.singles")}
+            valuePropName="checked"
+            rules={[
+              {
+                required: true,
+                message: i18n("CreateNcbiExport.files.required"),
+              },
+            ]}
+          >
+            <Checkbox.Group style={{ width: `100%` }}>
+              <Row>
+                {sample.files.singles.map((pair) => (
+                  <Col key={pair.key} span={24}>
+                    <Checkbox value={pair.id}>
+                      <Space>
+                        <Avatar
+                          size="small"
+                          style={{ backgroundColor: `var(--primary-grey)` }}
+                          icon={<SwapRightOutlined />}
+                        />
+                        {pair.name}
```

Comment:
We should probably add the file size and created date to the listing. This will require a change in the way the files are displayed. Applies to all the file listings (singles, pairs, etc that are used on this page)